core/math/linalg/glsl/linalg_glsl.odin

```diff
@@ -1356,6 +1356,76 @@ not :: proc{
 
 @(require_results) identity :: proc "c" ($M: typeid/matrix[$N, N]$T) -> M { return 1 }
 
+@(require_results)
+mat3Translate :: proc "c" (v: vec2) -> mat3 {
+	return {
+		1, 0, v.x,
+		0, 1, v.y,
+		0, 0, 1,
+   	}
+}
+@(require_results)
+mat3Scale :: proc "c" (v: vec2) -> mat3 {
+	return {
+		v.x, 0,   0,
+		0,   v.y, 0,
+		0,   0,   1,
+   	}
+}
+@(require_results)
+mat3Rotate :: proc "c" (radians: f32) -> mat3 {
+	c := cos(radians)
+	s := sin(radians)
+	return {
+		 c, s, 0,
+		-s, c, 0,
+		 0, 0, 1,
+	}
+}
```

Comment:
You've got the -ve sign on the wrong sine components, and this only rotates around one axis. 

This needs a better naming scheme to make it clear it is actually for 2D + 1 homogenous coordinate.